docs/demo_guide/README.md

```diff
@@ -26,8 +26,8 @@ Setup steps:
 3. Now you can play the rosbag:
 
     ```
-    sudo python docs/demo_guide/rosbag_helper.py demo_2.0.bag #download rosbag
-    rosbag play demo_2.0.bag --loop
+    sudo bash docs/demo_guide/rosbag_helper.sh download #download rosbag
+    rosbag play /apollo/docs/demo_guide/demo_2.0.bag --loop
```

Comment:
The file is at https://github.com/ApolloAuto/apollo/blob/master/docs/demo_guide/rosbag_helper.py

Perhaps are you using a branch?

Reply:
When I just cut paste it within teh docker env 

$ sudo python docs/demo_guide/rosbag_helper.py demo_2.0.bag
python: can't open file 'docs/demo_guide/rosbag_helper.py': [Errno 2] No such file or directory

$ pwd
/apollo

Reply:
$ ls /apollo/docs/demo_guide/
README.md  README_cn.md  demo.bag  demo_2.0.bag  images  rosbag_helper.sh

Reply:
Thx for your review .... I just cloned the master